src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -145,22 +146,22 @@ export default class GeBatchGiftEntryTable extends LightningElement {
                 section.elements
                     .filter(e => e.elementType === 'field')
                     .forEach(
-                    element => {
-                        const fieldWrapper = GeFormService.getFieldMappingWrapper(element.dataImportFieldMappingDevNames[0]);
-                        if (isNotEmpty(fieldWrapper)) {
-                            const column = {
-                                label: element.customLabel,
-                                fieldName: fieldWrapper.Source_Field_API_Name,
-                                type: GeFormService.getInputTypeFromDataType(
-                                    element.dataType
-                                ) === 'date' ? 'date-local' :
-                                    GeFormService.getInputTypeFromDataType(element.dataType)
-                            };
-
-                            this._columnsBySourceFieldApiName[column.fieldName] = column;
+                        element => {
```

Comment:
Just a heads up this block is changing based on the conflict resolution between Daniel's and my updates. Pinged you directly in slack.